src/coffee/sync/utils/product.coffee

```diff
@@ -624,6 +625,6 @@ actionsBaseList = ->
 
 allVariants = (product) ->
   {masterVariant, variants} = _.defaults product,
-    masterVariant: {}
+    masterVariant: undefined
     variants: []
   [masterVariant].concat variants
```

Comment:
Hey @butenkor 

yeah I tried that but with the way the code is setup, a masterVariant object is always expected to be in the product object and that it should be the first item in the variants array. So I set it to undefined and do not do any action on it [here](https://github.com/sphereio/sphere-node-sdk/pull/162/files#diff-a94247bc9925c68272e1e3c1213c3556R73)